roles/servicetelemetry/templates/manifest_smartgateway_metrics.j2

```diff
@@ -1,16 +1,39 @@
-apiVersion: smartgateway.infra.watch/v2alpha1
+apiVersion: smartgateway.infra.watch/v2
 kind: SmartGateway
 metadata:
   name: '{{ this_smartgateway }}'
   namespace: '{{ meta.namespace }}'
 spec:
-  amqpUrl: '{{ meta.name }}-interconnect.{{ meta.namespace }}.svc.cluster.local:5673/{{ this_collector.subscription_address }}'
-  amqpDataSource: '{{ this_collector.collector_type }}'
-  debug: {{ this_collector.debug_enabled | default(false) }}
-  serviceType: 'metrics'
-{% if this_collector.collector_type == "collectd" %}
+{% if this_collector.debug_enabled is defined and this_collector.debug_enabled %}
+  logLevel: "debug"
+{% else %}
+  logLevel: "info"
+{% endif %}
+  handleErrors: true
+{% if this_collector.collector_type is defined and this_collector.collector_type == "collectd" %}
   size: {{ smartgateway_deployment_size }}
```

Comment:
This reminds me that I need to run a deployment using HA mode to make sure it all still works.

Reply:
@csibbitt for events SmartGateways, do you remember off the top of your head whether we should be running a `size: 2` for those? Or is it only applicable to metrics?

If it's for everything, I think the fix I need to make is to drop the `collectd` filter for the smartgateway size parameter in the SG templates so that Ceilometer is also a size of 2. Everything is now sg-core + sg-bridge.

Reply:
So I set this to just follow the smartgateway_deployment_size across the board. Need to verify we're not regressing something by doing this, but since this is all sg-core now, it should be appropriate. The one thing that could cause an issue is because the sg-core was refactored, so need to verify that everything works.